articles/overview/overview.md

```diff
@@ -0,0 +1,67 @@
+author: bradben
+ms.author:  bradben
+ms.date: 04/22/2020
+ms.topic: overview
+uid: microsoft.quantum.overview.introduction
+---
+
+# Introduction to quantum computing 
+
+There are some problems so difficult, so incredibly vast, that even if every supercomputer in the world worked on the problem, it would still take longer than the lifetime of the universe to solve.
+
+Quantum computers hold the promise to solve some of our planet's biggest challenges - in environment, agriculture, health, energy, climate, materials science, and problems not yet even imagined. The impact of quantum computers will be far-reaching and have as great an impact as the creation of the transistor in 1947, which paved the way for today’s digital economy.
+
+## What is quantum computing?
+
+In modern usage, the word quantum means the _smallest possible discrete unit of any physical property_, usually referring to atomic or sub-atomic particles. Quantum computers use actual quantum particles – atoms or ions – as processing units, and are large, complex, and expensive devices. 
+
+Harnessing the unique behavior of quantum physics and applying it to computational computing, quantum computers introduce new concepts to traditional programming methods, making use of quantum physics behavior such as superposition, entanglement, and quantum interference. 
+
+## What can a quantum computer do?
+
+A quantum computer isn’t a supercomputer that can do everything faster, it just does some select things extremely well. 
+
+### Quantum simulation
+
+Since quantum computers use quantum particles as their processors, they are well-suited for modeling other quantum systems. Photosynthesis, superconductors, and complex molecules are examples of quantum systems that can be simulated using quantum programs.
+
+### Cryptography and Shor’s algorithm
+
+In 1994, Peter Shor showed that a scalable quantum computer can break every widely used encryption technique. Classical cryptography relies on the intractability of operations on large numbers, such as factorization of large numbers into two prime numbers, and quantum computers excel at factorization. 
+
+### Search and Grover’s algorithm
+
+In 1996, Lov Grover developed a quantum algorithm that dramatically sped up the solution to unstructured searches faster than any classical algorithm can do. This increased speed becomes significant in lists that contain millions or billions of items.
+
+### Optimization
+
+Optimization is the task of balancing a system for maximum efficiency and minimum cost. A quantum computer can speed up optimization algorithms and find solutions that otherwise were not possible. In addition to optimizing traffic flow to reduce congestion, there is airplane gate assignment, package delivery, job scheduling and more. With breakthroughs in materials science, there will be new forms of energy, batteries with greater capacity, lighter and stronger materials.
```

Comment:
```suggestion
Optimization is the task of balancing a system for maximum efficiency and minimum cost. A quantum computer can speed up optimization algorithms and find solutions that otherwise were not possible. In addition to optimizing traffic flow to reduce congestion, there is airplane gate assignment, package delivery, job scheduling and more. With breakthroughs in materials science, there will be new forms of energy, batteries with greater capacity, and lighter and stronger materials.
```